pkg/cloud-provider/cloudapi/azure/azure_crd_helpers.go

```diff
@@ -124,10 +124,7 @@ func ComputeVpcToInternalVpcObject(vnet *armnetwork.VirtualNetwork, namespace st
 			cidrs = append(cidrs, *cidr)
 		}
 	}
-	labelsMap := map[string]string{
-		common.VpcLabelAccountName: accountName,
```

Comment:
Why is labels removed?

Reply:
Since the labels are common for both AWS and Azure, moved it to crd_generator.go.